mitosheet/src/utils/formatColumns.tsx

```diff
@@ -1,346 +0,0 @@
-import React from "react"
-import MitoAPI from "../jupyter/api"
-import DropdownItem from "../components/elements/DropdownItem"
-import { getSelectedNumberSeriesColumnIDs } from "../components/endo/selectionUtils"
-import { ColumnID, FormatType, FormatTypeObj, GridState, MitoSelection, SheetData } from "../types"
-import DropdownCheckmark from '../components/icons/DropdownCheckmark'
-import { isNumberDtype } from "./dtypes"
-
-export const FORMAT_DISABLED_MESSAGE = 'You must have at least one Number column selected to adjust the formatting.'
-
-
-const formatCellDataAsStringWithCommas = (cellData: string | number | boolean, decimalPlaces?: number): string => {
-    return Number(cellData).toLocaleString("en-US", {minimumFractionDigits: decimalPlaces, maximumFractionDigits: decimalPlaces})
-}
-
-/*
-    For a cell to be formatted as a number, the cell must only contain valid number symbols
-    and be either a int or float column. 
-
-    Note: The cellData type of a float is a string.
-*/
-export const displayCellAsNumber = (cellData: boolean | string | number, columnDtype: string): boolean => {
-    // If the column is not a number series, then don't format the cell as a number
-    if (!isNumberDtype(columnDtype)) {
-        return false
-    }
-
-    if (typeof cellData === 'boolean') {
-        return false
-    } else if (typeof cellData === 'number') {
-        return true
-    } else {
-        return isStringNumeric(cellData)
-    }
-}
-
-/*
-    Determines if a string is a number
-    Adapted from: https://stackoverflow.com/questions/175739/how-can-i-check-if-a-string-is-a-valid-number
-    Note: This won't format numbers that have non . separators correctly, but that's okay for now because we wouldn't want to switch them to decimals
-*/
-export const isStringNumeric = (str: string): boolean => {
```

Comment:
Similar to the comment above, this code can all disappear! We drop a lot of complexity by just making sure the data we're sending the frontend is in a format that is easy for us to consume!